lib/charms/docker/compose.py

```diff
@@ -0,0 +1,46 @@
+import os
+
+from contextlib import contextmanager
+from shlex import split
+from subprocess import check_call
+
+# Wrapper and convenience methods for charming w/ docker compose in python
+class Compose:
+    def __init__(self, workspace):
+        self.workspace = workspace
+        self.validate_workspace()
+
+    def start_service(self, service=None):
```

Comment:
we should name these methods similar to the actual docker-compose command.
Suggestion:  def up(self, service=None):